index.bs

```diff
@@ -1503,6 +1583,36 @@ filter only those that ought to be returned to the local end.
 
 </div>
 
+### Types ### {#module-session-types}
+
+#### The session.CapabilitiesRequest Type #### {#type-session-capabilitiesRequest}
+
+[=remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+Capabilities = {
+    ?acceptInsecureCertificates: bool,
+    ?browserName: text,
+    ?browserVersion: text,
+    ?platformName: text,
+    ?proxy: {
+        ?proxyType: "pac" / "direct" / "autodetect" / "system" / "manual",
+        ?proxyAutoconfigUrl: text,
+        ?ftpProxy: text,
+        ?httpProxy: text,
+        ?noProxy: [*text],
+        ?sslProxy: text,
+        ?socksProxy: text,
+        ?socksVersion: int,
+    },
+    *text => any
```

Comment:
What about `setWindowRect`?

Reply:
I believe that only appears in the response; it doesn't seem to be something that's handled as part of the request. Maybe it should be?

Reply:
I'm not exactly sure how the capability matching works from the client side. There might be cases when there is a requirement for tests to actually enforce moving and resizing of windows. I doubt this could be covered by just the platform capability? There might still be window managers (?) that on desktop that don't allow it?

But this would also require a change in the WebDriver HTTP spec, so maybe we leave it as is (at least for now).

Reply:
Yes, I think that's out of scope for this issue. If there's a problem here let's file an issue on the WebDriver classic spec.